src/lti/LTI_Constants.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+namespace IMSGlobal\LTI;
+
+class LTI_Constants
+{
+    public const V1_3 = '1.3.0';
+
+    // Required message claims
+    public const MESSAGE_TYPE = 'https://purl.imsglobal.org/spec/lti/claim/message_type';
+    public const VERSION = 'https://purl.imsglobal.org/spec/lti/claim/version';
+    public const DEPLOYMENT_ID = 'https://purl.imsglobal.org/spec/lti/claim/deployment_id';
+    public const TARGET_LINK_URI = 'https://purl.imsglobal.org/spec/lti/claim/target_link_uri';
+    public const RESOURCE_LINK = 'https://purl.imsglobal.org/spec/lti/claim/resource_link';
+    public const ROLES = 'https://purl.imsglobal.org/spec/lti/claim/roles';
+
+    // Optional message claims
+    public const CONTEXT = 'https://purl.imsglobal.org/spec/lti/claim/context';
+    public const TOOL_PLATFORM = 'https://purl.imsglobal.org/spec/lti/claim/tool_platform';
+    public const ROLE_SCOPE_MENTOR = 'https://purlimsglobal.org/spec/lti/claim/role_scope_mentor';
+    public const LAUNCH_PRESENTATION = 'https://purl.imsglobal.org/spec/lti/claim/launch_presentation';
+    public const LIS = 'https://purl.imsglobal.org/spec/lti/claim/lis';
+    public const CUSTOM = 'https://purl.imsglobal.org/spec/lti/claim/custom';
+
+    // LTI DL
+    public const DL_CONTENT_ITEMS = 'https://purl.imsglobal.org/spec/lti-dl/claim/content_items';
+    public const DL_DATA = 'https://purl.imsglobal.org/spec/lti-dl/claim/data';
+    public const DL_DEEP_LINK_SETTINGS = 'https://purl.imsglobal.org/spec/lti-dl/claim/deep_linking_settings';
+
+    // LTI NRPS
+    public const NRPS_NAMESROLESPROVISIONINGSERVICE = 'https://purl.imsglobal.org/spec/lti-nrps/claim/namesroleservice';
```

Comment:
Naming is always a pain, it would be clearer if we did `<SERVICE>_<TYPE>_<NAME>` e.g. `NRPS_CLAIM_SERVICE` and `NRPS_SCOPE_READ_ONLY`

Reply:
Sounds good. I'll implement this as soon as I get a chance.